### ISSUE TITLE
Building deb package fails: dh_clean: Compatibility levels before 5 are no longer supported (level 4 requested)

### ISSUE THREAD

**Issue by diqidoq** [hydrogen-music/hydrogen#493]
Compiling from source works only with symlinks for raptor.h (issue #342 my last [comment](https://github.com/hydrogen-music/hydrogen/issues/342#issuecomment-293045972)) and building deb package fails because of 

    me@hydrogen/linux: sudo fakeroot dpkg-buildpackage

    dpkg-buildpackage: info: source package hydrogen
    dpkg-buildpackage: info: source version 1.0.0-alpha
    dpkg-buildpackage: info: source distribution unstable
    dpkg-buildpackage: info: source changed by Alessandro Cominu <comix@users.sf.net>
    dpkg-buildpackage: info: host architecture amd64
    dpkg-source --before-build linux
    debian/rules clean
    dh_testdir
    dh_testroot
    rm -f build-stamp configure-stamp
    find . -name '*.py[co]' | xargs rm -f
    # Add here commands to clean up after the build process.
    rm -rf ../builddebian;
    dh_clean: Compatibility levels before 5 are no longer supported (level 4 requested)
    debian/rules:49: recipe for target 'clean' failed
    make: *** [clean] Error 25 
    dpkg-buildpackage: error: debian/rules clean gave error exit status 2

**Comment by jdekozak** [comment]
Hi,
I cannot reproduce the build failure on Debian 8.7 with current master branch.
What is your distribution version ? Maybe someone else can help you...
Regards,

**Comment by mjsuarez** [comment]
I ran into this same issue on Linux Mint 18 Sarah (based on Ubuntu Xenial). After some searching around I came across the suggestion to change the value in my hydrogen/linux/debian/compat from 4 to 9 before running build.sh, and now fakeroot successfully builds the deb.

I can't vouch for this being a "correct" solution since I generally don't know what I'm doing, but if it helps until someone with more knowledge than me comes along, so be it.

Good Luck
